zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementFacade.kt

```diff
@@ -185,6 +191,22 @@ class ObjectManagementFacade(
         }
     }
 
+    fun deleteObject(
+        objectName: String,
+        objectId: UUID
+    ): HttpStatus {
+        val accessObject = getAccessObject(objectName)
+
+        try {
+            logger.trace { "Deleting object '$objectId' of type '${accessObject.objectManagement.objecttypeId}' from Objecten API using plugin ${accessObject.objectManagement.objectenApiPluginConfigurationId}" }
+            return accessObject.objectenApiPlugin.deleteObject(
+                URI("${accessObject.objectenApiPlugin.url}objects/$objectId")
+            )
+        } catch (ex: HttpClientErrorException) {
+            throw HttpClientErrorException(ex.statusCode, "Error while deleting object $objectId. Response from Objects API: ${ex.responseBodyAsString}")
```

Comment:
Please pass the old exception to the new exception so that the entire stacktrace is saved. Like this:
```suggestion
            throw IllegalStateException("Error while deleting object $objectId. Response from Objects API: ${ex.responseBodyAsString}", ex)
```